hw/ip/csrng/data/csrng.hjson

```diff
@@ -0,0 +1,170 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "csrng",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  param_list: [
+    { name: "NHwApps",
+      type: "int",
+      default: "4",
+      desc: "Number of CSRNG hardware applications",
+      local: "true"
+    } 
+  ],
+  interrupt_list: [
+    { name: "cs_cmd_req_done"
+      desc: "Asserted when a command request is completed."}
+    { name: "cs_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ],
+  inter_signal_list: [
+    { struct:  "csrng_req",
+      type:    "req_rsp",
+      name:    "csrng_req",
+      act:     "requester",
+      package: "csrng_pkg",
+    }
+    { struct:  "csrng_rsp",
+      type:    "req_rsp",
+      name:    "csrng_rsp",
+      act:     "responder",
+      package: "csrng_pkg",
```

Comment:
You only need to define `csrng` if type is `req_rsp`. then the topgen will make `csrng_req` `csrng_rsp` both.

Reply:
Also, Unfortunately, parameterized array of struct in inter_module signal isn't yet supported. Sorry for that, but temporarily, could you please add `width` with integer value here?

Reply:
understand the req_rsp change - done. Do not understand the parameterized array of struct issue. Can you point to exactly what you are saying?

Reply:
```verilog
  // Application Interfaces
  input  csrng_req_t  [NHwApps:1] csrng_cmd_i,
  output csrng_rsp_t  [NHwApps:1] csrng_cmd_o,
```
Above interface in `csrng.sv` uses `NHwApps` parameter in the parameter port list. Though this is not supported in inter-module yet. So you need to add `width: 3` (if HwApps is 3) to the inter-module data structure.